project/src/components/reviews/reviews-list.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import reviewsType from '../reviews-props/reviews-props.js';
+import ReviewsItem from '../reviews/reviews-item.jsx';
+
+function ReviewsList(props) {
+  const { reviews } = props;
+  const getReviewsCout = () => reviews.length;
+  const reviewsCout = getReviewsCout();
```

Comment:
```suggestion
  const reviewsCount = reviews.length;
```
Лишние вызовы не нужны)